Makefile

```diff
@@ -97,5 +97,5 @@ test:
 check:
 	go fmt ./...
 	go vet ./...
-	go mod tidy -compat=1.17
-	docker run --rm -v ${PWD}:/app -w /app golangci/golangci-lint:v1.50.1 golangci-lint run
+	go mod tidy
+	docker run --rm -v ${PWD}:/app -w /app golangci/golangci-lint:v1.50.0 golangci-lint run
```

Comment:
why did you change `golangci-lint` to a previous version?

Reply:
It a mistake after rebase. Fixed. Thanks